doc/source/enhancingperf.rst

```diff
@@ -213,17 +213,16 @@ the rows, applying our ``integrate_f_typed``, and putting this in the zeros arra
 
 .. warning::
 
-   In 0.13.0 since ``Series`` has internaly been refactored to no longer sub-class ``ndarray``
-   but instead subclass ``NDFrame``, you can **not pass** a ``Series`` directly as a ``ndarray`` typed parameter
+   You can **not pass** a ``Series`` directly as a ``ndarray`` typed parameter
    to a cython function. Instead pass the actual ``ndarray`` using the ``.values`` attribute of the Series.
 
-   Prior to 0.13.0
+   So, do not do this:
```

Comment:
maybe:

This will fail because the cython definition is specific to an ``ndarray`` and not the passed ``Series``